longfi-us915/longfi-us915.ino

```diff
@@ -232,11 +241,46 @@ void do_send(osjob_t* j){
     if (LMIC.opmode & OP_TXRXPEND) {
         Serial.println(F("OP_TXRXPEND, not sending"));
     } else {
-        // Prepare upstream data transmission at the next possible time.
-        LMIC_setTxData2(1, mydata, sizeof(mydata)-1, 0);
+        
+       // Prepare upstream data transmission at the next possible time.
+        static uint8_t payload[32];
+        uint8_t idx = 0;
+        uint32_t data;
+
+        if (GPS.newNMEAreceived()) {
+          GPS.parse(GPS.lastNMEA());
+        }
+        
+        if (GPS.fix) {
+          Serial.println(GPS.latitudeDegrees);
+          Serial.println(GPS.longitudeDegrees);
+          //data = GPS.latitude_fixed * (GPS.lat == 'N' ? 1 : -1) + 90 * 1E7;
```

Comment:
What's this about? It look if we throw away the logic from these comments that we will have trouble running in Southern longitudes? It looks like the latitude logic needs help too.